besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -909,7 +909,7 @@ void setBannedNodeIds(final List<String> values) {
   @Option(
       names = {"--target-gas-limit"},
       description =
-          "Sets target gas limit per block. If set each blocks gas limit will approach this setting over time if the current gas limit is different.")
+          "Sets target gas limit per block. If set each block's gas limit will approach this setting over time if the current gas limit is different.")
```

Comment:
"set, each" ?

Reply:
this sort of thing appears in the help both ways. If we're going to make it consistent that would be a bigger change than this PR.